packages/t/tiled/package.yml

```diff
@@ -13,14 +14,24 @@ component  : multimedia.graphics
 summary    : Tiled is a general purpose tile map editor.
 description: |
     Tiled is a general purpose tile map editor.
+libsplit   : no
+environment: |
+    export QBS=$workdir/qbs-2.3.0/bin/qbs
 builddeps  :
     - pkgconfig(Qt5Help)
     - pkgconfig(Qt5Qml)
+    - pkgconfig(Qt5Svg)
     - pkgconfig(python3)
 setup      : |
+    tar xf $sources/v2.3.0.tar.gz
+    pushd qbs-2.3.0
+        %cmake .
```

Comment:
Use `%cmake_ninja` instead of `%cmake`. This also means changing `%make` to `%ninja_build`

Reply:
I'll change it for 1.10.3. Anything else?